ir/visitor.h

```diff
@@ -287,6 +314,11 @@ class ControlFlowVisitor : public virtual Visitor {
     virtual ControlFlowVisitor *clone() const = 0;
     void init_join_flows(const IR::Node *root) override;
     bool join_flows(const IR::Node *n) override;
+
+    /** @returns true if this node should be removed from the set of join
+     * points, where a 'join point' is a node with multiple incoming edges in
+     * the control flow graph.
```

Comment:
This will be called for all nodes with multiple incoming edges, and should return 'false' if the node should be considered a join point, and 'true' if if should not be considered one.  Nodes with only one incoming edge are never join points.